master/internal/db/postgres_filters.go

```diff
@@ -60,6 +60,10 @@ func filterToSQL(
 		return fmt.Sprintf("AND %s > $%d", field, paramID)
 	case api.FilterOperationLessThanEqual:
 		return fmt.Sprintf("AND %s <= $%d", field, paramID)
+	case api.FilterOperationStringContainment:
+		// Suprisingly (?) this work for both bytea (needed for current)
+		// as well as plain text fields (not needed atm)
+		return fmt.Sprintf("AND encode(%s::bytea, 'escape') ILIKE  ('%%%%' || $%d || '%%%%')", field, paramID)
```

Comment:
hilarous code

Reply:
[related](https://github.com/determined-ai/determined/pull/4270) to the hilarious code